app/src/main/java/com/github/sdp_begreen/begreen/models/meetings/Meeting.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.sdp_begreen.begreen.models.meetings
+
+import android.os.Parcelable
+import com.github.sdp_begreen.begreen.models.CustomLatLng
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Class that represents a waist collection meeting
+ */
+@Parcelize
+data class Meeting(
+    var meetingId: String? = null,
+    var creator: String? = null,
+    var title: String? = null,
+    var description: String? = null,
+    var startDateTime: Long? = null,
+    var endDateTime: Long? = null,
+    var startCoordinates: CustomLatLng? = null,
+    var endCoordinates: CustomLatLng? = null,
+    var intermediaryCoordinates: List<CustomLatLng>? = null,
```

Comment:
Actually I did it like that because of the way the API to google map was done in order to retrieve a `Direction` given an `origin` and a `destination` and some intermediary points. A call looks like that:
```
https://maps.googleapis.com/maps/api/directions/json?origin=46.833801, 7.074858&destination=46.80715315997017, 7.152001373012503&mode=walking&units=metric&waypoints=via:46.806024, 7.096920|via:46.790383, 7.117481|via:46.794971, 7.150909
```

We could have an array yes, but I find it make more sens to call `.startCoordinates` for `origin` rather than `coordinates[0]`. But That's arguable. What do you think ?

Reply:
Oh okay, that's fine then